region/norfair/crocomire/Grapple Tutorial Room 3.json

```diff
@@ -560,12 +638,24 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [3, 2],
+      "name": "Go Through Missing Gate",
+      "requires": [
+        {"obstaclesCleared": ["B"]}
+      ],
+      "resetsObstacles": ["B"]
+    },
     {
       "link": [3, 4],
       "name": "Base",
       "requires": [
-        {"obstaclesCleared": ["A"]}
-      ]
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          {"obstaclesCleared": ["B"]}
```

Comment:
This strat is a run and jump using the runway from the gate being gone.
[2,3] resets[B], So its more like you would want a [24] copy for the GMode case.

Reply:
Its possible putting `Resets[B]` on strats which `Clear[A]` is safer too.

You should be able to run and jump from 2 before ending GMode.  But you cannot Farm while in GMode and make it back to a despawned gate.